specification/portal/Dashboard.Management/Dashboard.tsp

```diff
@@ -0,0 +1,78 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@azure-tools/typespec-client-generator-core";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+
+using Azure.ClientGenerator.Core;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+using TypeSpec.Rest;
+
+namespace Microsoft.Portal;
+/**
+ * The shared dashboard resource definition.
+ */
+@doc("The shared dashboard resource definition.")
+model Dashboard
+  is Azure.ResourceManager.TrackedResource<DashboardPropertiesWithProvisioningState> {
+  ...ResourceNameParameter<
+    Resource = Dashboard,
+    KeyName = "dashboardName",
+    SegmentName = "dashboards"
+  >;
+}
+
+@armResourceOperations
+interface Dashboards {
+  /**
+   * Gets the Dashboard.
+   */
+  get is ArmResourceRead<Dashboard>;
+
+  /**
+   * Creates or updates a Dashboard.
+   */
+  createOrUpdate is ArmResourceCreateOrReplaceSync<Dashboard>;
+
+  /**
+   * Updates an existing Dashboard.
+   */
+  @parameterVisibility
+  update is ArmCustomPatchSync<Dashboard, PatchableDashboard>;
+
+  /**
+   * Deletes the Dashboard.
+   */
+  delete is ArmResourceDeleteSync<Dashboard>;
```

Comment:
Should use `ArmResourceDeleteWithoutOkSync`

Reply:
Do you mean `ArmResourceDeleteWithoutOkAsync`? But the existing one is not `async`. Thus, keeping it as it is.